tendermint/src/vote.rs

```diff
@@ -65,7 +68,7 @@ impl TryFrom<RawVote> for Vote {
 
     fn try_from(value: RawVote) -> Result<Self, Self::Error> {
         if value.timestamp.is_none() {
-            return Err(NoTimestamp.into());
+            return Err(error::no_timestamp_error(anyhow::anyhow!("no timestamp error")));
```

Comment:
This can be simplified into just

```rust
return Err(error::no_timestamp_error());
```

Behind the scene `flex-error` automatically calls `anyhow` to capture the stack trace already.

Reply:
I have reset this change.